notebooks/DiCE_getting_started_feasible.ipynb

```diff
@@ -0,0 +1,1072 @@
+       "3              40       0  \n",
+       "4              40       0  "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dataset.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'age': 'age',\n",
+       " 'workclass': 'type of industry (Government, Other/Unknown, Private, Self-Employed)',\n",
+       " 'education': 'education level (Assoc, Bachelors, Doctorate, HS-grad, Masters, Prof-school, School, Some-college)',\n",
+       " 'marital_status': 'marital status (Divorced, Married, Separated, Single, Widowed)',\n",
+       " 'occupation': 'occupation (Blue-Collar, Other/Unknown, Professional, Sales, Service, White-Collar)',\n",
+       " 'race': 'white or other race?',\n",
+       " 'gender': 'male or female?',\n",
+       " 'hours_per_week': 'total work hours per week',\n",
+       " 'income': '0 (<=50K) vs 1 (>50K)'}"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# description of transformed features\n",
+    "adult_info = helpers.get_adult_data_info()\n",
+    "adult_info"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Given this dataset, we construct a data object for DiCE. Since continuous and discrete features have different ways of perturbation, we need to specify the names of the continuous features. DiCE also requires the name of the output variable that the ML model will predict."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d = dice_ml.Data(dataframe=dataset, continuous_features=['age', 'hours_per_week'], outcome_name='income')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Loading the ML model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Below, we use a pre-trained ML model which produces high accuracy comparable to other baselines. For convenience, we include the sample trained model with the DiCE package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ML Model Sequential(\n",
+      "  (0): Linear(in_features=29, out_features=20, bias=True)\n",
+      "  (1): ReLU()\n",
+      "  (2): Linear(in_features=20, out_features=2, bias=True)\n",
+      "  (3): Softmax(dim=None)\n",
+      ")\n"
+     ]
+    }
+   ],
+   "source": [
+    "backend = 'PYT'\n",
+    "ML_modelpath = helpers.get_adult_income_modelpath(backend=backend)\n",
+    "ML_modelpath = ML_modelpath[:-4] + '_2nodes.pth'\n",
+    "m = dice_ml.Model(model_path= ML_modelpath, backend=backend)\n",
```

Comment:
you might have to change the backend variable to how it is suggested here https://github.com/interpretml/DiCE/blob/master/dice_ml/model.py#L13